.env.alfajores

```diff
@@ -12,3 +12,4 @@ IOS_GOOGLE_SERVICE_PLIST=GoogleService-Info.alfajores.plist
 SENTRY_ENABLED=true
 AUTH0_DOMAIN=auth.alfajores.valora.xyz
 ONBOARDING_FEATURES_ENABLED=CloudBackupSetup,CloudBackupRestore,CloudBackupInOnboarding,EnableBiometry,ProtectWallet
+DEEP_LINK_PREFIX=celo
```

Comment:
What do you think of naming it `DEEP_LINK_URL_SCHEME`?
It better represent what it uses and more closely follows what iOS and Android use for this.